plugins/inventory/aws_ec2.py

```diff
@@ -133,6 +133,12 @@
       - The suffix for host variables names coming from AWS.
     type: str
     version_added: 3.1.0
+  use_ssm_inventory:
+    description:
+      - Add SSM inventory information into hostvars.
+    type: bool
+    default: False
+    version_added: 5.3.0
```

Comment:
```suggestion
    version_added: 6.0.0
```

This won't cleanly backport, there's some substantial changes between 5.x and 6.x that I wasn't comfortable backporting.